src/tls.c

```diff
@@ -764,7 +764,8 @@ static void tlsEventHandler(struct aeEventLoop *el, int fd, void *clientData, in
 }
 
 static void tlsAcceptHandler(aeEventLoop *el, int fd, void *privdata, int mask) {
-    int cport, cfd, max = MAX_ACCEPTS_PER_CALL;
```

Comment:
i see you didn't make a similar change in unix.c, and didn't delete the define in connection.h.
is that on purpose?
considering that, i'd expected to see some detail about that in the docs redis.conf

Reply:
it feels odd to have that constant (MAX_ACCEPTS_PER_CALL).
maybe hard code it here with some comment?

Reply:
You are right, as the major impact is with TLS, the change focused on it. 
As @JimB123 mentioned, as the event loop spins fast enough, changing both should be OK.
I can expend the change (and remove the use of MAX_ACCEPTS_PER_CALL entirely) if you prefer.

Reply:
Please do. I think it's less confusing